packages/formstr-app/src/containers/CreateFormNew/components/AnswerSettings/index.tsx

```diff
@@ -25,6 +27,17 @@ function AnswerSettings() {
       option.answerSettings.renderElement === answerSettings.renderElement
   );
 
+  const [uploadedImages, setUploadedImages] = useState<Array<{name: string, url: string}>>(() => {
```

Comment:
why do we need this state when we are already using the editQuestion context?

Reply:
i was toying with the idea of rendering multiple images on the right hand side, and adding a delete button there. i added it then. later removed the rendering feature, forgot to remove this. making the changes now!